### PR TITLE
feat: Add a little avatar of the sender on the corner of a group icon in chat list

### PR DESCRIPTION
**Pull request by Clansty**
It seems like this:

image

### CODE REVIEW

**Comment by mochaaP** [comment]
Could you add an option in "Enhanced Settings" for this?

**Comment by Clansty** [comment]
will try

**Comment by Clansty** [comment]
> Could you add an option in "Enhanced Settings" for this?

Added

**Comment by c0re100** [comment]
Thanks.

File: Telegram/SourceFiles/core/enhanced_settings.cpp [hunk 1]
```diff
@@ -368,6 +369,7 @@ namespace EnhancedSettings {
 		settings.insert(qsl("disable_link_warning"), GetEnhancedBool("disable_link_warning"));
 		settings.insert(qsl("blocked_user_spoiler_mode"), GetEnhancedBool("blocked_user_spoiler_mode"));
 		settings.insert(qsl("disable_premium_animation"), GetEnhancedBool("disable_premium_animation"));
+		settings.insert(qsl("show_group_sender_avatar"), GetEnhancedBool("disable_premium_animation"));
```
Comment (mochaaP) [inline Telegram/SourceFiles/core/enhanced_settings.cpp:372]: @c0re100 typo here

Comment (clansty) [reply Telegram/SourceFiles/core/enhanced_settings.cpp:372]: 😭

Comment (c0re100) [reply Telegram/SourceFiles/core/enhanced_settings.cpp:372]: https://github.com/TDesktop-x64/tdesktop/commit/92a684f3ad441d8109600ed2352e3e0feb773d13

**Pull request merged**